src/components/views/messages/TextualBody.js

```diff
@@ -80,6 +93,20 @@ module.exports = React.createClass({
                     }
                 }, 10);
             }
+            const buttons = ReactDOM.findDOMNode(this).getElementsByClassName("mx_EventTile_copyButton");
+            if (buttons.length > 0) {
+                // Do this asynchronously: parsing code takes time and we don't
+                // need to block the DOM update on it.
+                setTimeout(() => {
+                    if (this._unmounted) return;
+                    for (let i = 0; i < buttons.length; i++) {
+                        buttons[i].onclick = (e) => {
+                            const copyCode = buttons[i].parentNode.getElementsByTagName("code")[0];
+                            this.copyToClipboard(copyCode.textContent);
+                        };
+                    }
+                }, 10);
+            }
```

Comment:
This looks like the right approach, although I don't understand why you're adding the onclick handlers asynchronously afterwards here. You've also copied the comment from the block above, but it's not relevant to this block and quite misleading.